src/components/SearchBar/SearchBar.js

```diff
@@ -181,7 +183,10 @@ SearchBar.defaultProps = {
   placeholder: 'Search',
   searchIcon: <SearchIcon style={{ color: grey[500] }} />,
   style: null,
-  value: ''
+  value: '',
+  onChange: noop,
```

Comment:
I'm not sure if `noop` as default value is better than just checking if the prop is defined. :thinking: It's shorter for sure.